glowing_waffles/notebooks/acp.py

```diff
@@ -0,0 +1,107 @@
+class ObserveACP:
+    
+    def __init__(self, name, date):
+        
+        from astropy.coordinates import SkyCoord
+        from astropy.time import Time
+        import datetime
+    
+        self.name = name
+        self.filters = []
+        self.binning = 1
+        
+        self.ra = SkyCoord.from_name(name).ra.deg
+        self.dec = SkyCoord.from_name(name).dec.deg
+        
+        #self.date = str(datetime.datetime.today()).split()[0]
+        self.date = date
+        
+        
+    def wait_until(self, time):
+        self.startTime = time
```

Comment:
A couple comment on this:

+ snake_case for attributes too, so `self.start_time`
+ You should probably set a default `start_time` in `__init__` so that the attribute `self.start_time` always exists. Otherwise, creating an `ObserveACP` and then writing will result in a failure because the attribute `start_time` won't exist.